From 5a62944a6801f0da0a67b986002981442766614c Mon Sep 17 00:00:00 2001
From: Felix Fietkau <nbd@nbd.name>
Date: Tue, 27 Sep 2022 16:22:06 +0200
Subject: [PATCH 66/96] kernel: add a few patches for avoiding unnecessary skb
 reallocations - significantly improves ethernet<->wireless performance

lede-commit: 6f89cffc9add6939d44a6b54cf9a5e77849aa7fd
Signed-off-by: Felix Fietkau <nbd@nbd.name>
---
 include/linux/skbuff.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/skbuff.h b/include/linux/skbuff.h
index 63d0a21b6316..ec3d3246c7a7 100644
--- a/include/linux/skbuff.h
+++ b/include/linux/skbuff.h
@@ -3123,7 +3123,7 @@ static inline int pskb_network_may_pull(struct sk_buff *skb, unsigned int len)
  * NET_IP_ALIGN(2) + ethernet_header(14) + IP_header(20/40) + ports(8)
  */
 #ifndef NET_SKB_PAD
-#define NET_SKB_PAD	max(32, L1_CACHE_BYTES)
+#define NET_SKB_PAD	max(64, L1_CACHE_BYTES)
 #endif
 
 int ___pskb_trim(struct sk_buff *skb, unsigned int len);
-- 
2.37.2

